msgraph-mail/go/utilities/models/body_type.go

```diff
@@ -1,6 +1,5 @@
 package models
 import (
```

Comment:
@andrueastman can you refresh with https://github.com/microsoft/kiota/pull/4651 when you have a couple of minutes please?